docs/styles/transitions.md

```diff
@@ -72,6 +76,75 @@ Instead of this
 }
 ```
 
+#### Styling the overlay and content
+
+It is possible to independently style the transitions for the modal 
+overlay and modal content. 
+
+Keep in mind that inline styles will take precedence over those defined 
+as CSS rules. If you want to apply a transition to one of the default styles,
+such as `transform`, you can remove it from the inline definition and define
+it instead as a CSS rule.
```

Comment:
```suggestion
Keep in mind that inline styles will take precedence over those defined 
as CSS rules. In case you need to use both at the same time, you'll need to use the css `!important`.
```

Reply:
Just made more general because it will happen to other properties as well.